### ISSUE TITLE
There should be no limit on number of Conditions

### ISSUE THREAD

**Issue by pnickerson-cashstar** [aws-cloudformation/cfn-lint#3957]
### CloudFormation Lint Version

1.22.5

### What operating system are you using?

Fedora Linux 41 (Workstation Edition)

### Describe the bug

In `src/cfnlint/data/schemas/other/conditions/conditions.json`, `maxProperties` is set to 200. But Amazon does not actually impose any limit on the Conditions count:
- https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cloudformation-limits.html
- https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/conditions-section-structure.html

Currently, running cfn-lint with more than 200 Conditions results in:
```
E8001 expected maximum property count: 200, found: ###
***.yaml:###:1
```
The line number in the YAML file brings me right to the Conditions declaration.

### Expected behavior

cfn-lint does not raise rule E8001.

### Reproduction template

```yaml
AWSTemplateFormatVersion: 2010-09-09
Parameters:
  EnvType:
    Type: String
    Default: ''
Conditions:
  EnvTypeExists001: !Not [ !Equals [ !Ref EnvType, '' ] ]
  EnvTypeExists002: !Not [ !Equals [ !Ref EnvType, '' ] ]
  ...
  EnvTypeExists200: !Not [ !Equals [ !Ref EnvType, '' ] ]
  EnvTypeExists201: !Not [ !Equals [ !Ref EnvType, '' ] ]
Resources:
  Bucket001:
    Type: 'AWS::S3::Bucket'
    Condition: EnvTypeExists001
  Bucket002:
    Type: 'AWS::S3::Bucket'
    Condition: EnvTypeExists002
  ...
  Bucket200:
    Type: 'AWS::S3::Bucket'
    Condition: EnvTypeExists200
  Bucket201:
    Type: 'AWS::S3::Bucket'
    Condition: EnvTypeExists201
```
